webapp/channels/src/components/admin_console/access_control/editors/table_editor/operator_selector_menu.tsx

```diff
@@ -164,72 +167,65 @@ export default OperatorSelectorMenu;
 
 const getOperatorDescriptor = (operatorValue: string): OperatorDescriptor => {
     for (const descriptor of Object.values(OPERATOR_DESCRIPTORS)) {
-        if (descriptor.operatorValue === operatorValue) {
+        if (descriptor.id === operatorValue) {
             return descriptor;
         }
     }
 
     return OPERATOR_DESCRIPTORS.is;
 };
 
-type OperatorID = 'is' | 'is_not' | 'in' | 'starts_with' | 'ends_with' | 'contains';
+type OperatorID = OperatorLabel.IS | OperatorLabel.IS_NOT | OperatorLabel.IN | OperatorLabel.STARTS_WITH | OperatorLabel.ENDS_WITH | OperatorLabel.CONTAINS;
```

Comment:
This type isn't needed any more since it's actually just `OperatorLabel`. Interestingly, if you mouse over `OperatorID` in VS Code, it actually reduces this line to `type OperatorID = OperatorLabel`